### DISCUSSION TITLE
Different PnP Display Use

### DISCUSSION THREAD

**Discussion by tadgill** [SmoothWAN/SmoothWAN#60] [General & Help]
I have a 128x64 i2c that is .96" that works as intended (great feature)! 

I also ordered this: "[MakerFocus OLED Display Module I2C 128X64 1.3 Inch Display SSD1106](https://smile.amazon.com/dp/B08V97FYD2?ref=ppx_yo2ov_dt_b_product_details&th=1)" when I try it I see like one line of the SmoothWAN status then "FUZZ" or "Snow" on the rest of the screen. 

What do I need to change to make the larger screen work?

**Comment by TalalMash** [comment]
Hello @tadgill !
Have you confirmed that the display works with an Arduino/Pi app?
The SSD1106 are known to have QC issues, I can send a custom OLED app to test in SmoothWAN in case it works well.

**Reply by tadgill** [reply]
I only confirmed that the smaller .96 one works then shutdown, plugged the 1.3" SSD1106 in and started up. I bought a two pack and they both show the exact same output.

**Reply by TalalMash** [reply]
Great, will compile a new `ssd` tool and upload it here when ready, I'll move the pixels to 132 range. I'm assuming that it's 3/4 bottom area that is flaky?

**Reply by tadgill** [reply]
Seems like its all "off" but it is fuzzy more than 90% The one I soldered in is the same so it is not the connection.

![IMG_1651](https://user-images.githubusercontent.com/6070932/185712386-6abe699c-99f5-4dfa-ae13-0a23d2ce654a.jpeg)

**Reply by TalalMash** [reply]
Replace `/usr/bin/ssd` with https://github.com/TalalMash/SSD1306_OLED_json/raw/sh1106/sh1106_arm64 (use wget and cp)
Rename to `ssd` then reboot.
May need to center the display, post a pic to be sure.
